.github/workflows/merge-conflicts.yml

```diff
@@ -11,5 +11,5 @@ jobs:
     steps:
       - uses: mschilde/auto-label-merge-conflicts@master
```

Comment:
We should use the fixed version for this

Reply:
I second this (as it is best practice) + your dependabot config will already auto update the actions when there are new versions